README.md

```diff
@@ -6,6 +6,20 @@ Che docs use Jekyll to convert `.adoc` files into HTML pages. Docs are published
 
 There is a `run.sh` script in the root of the repo that runs a Docker image, mounts sources, and starts Jekyll. When running locally, docs are available at `localhost:4000`. Jekyll watches for changes in `.adoc` files and re-generates resources, so you can preview changes live.
 
+### Building natively
+
+If you prefer to not use a Docker image, or you're already in a container and can't nest, you instead need to [install these jekyll dependencies](https://jekyllrb.com/docs/installation/):
+
+```
+sudo yum install maven ruby ruby-devel @development-tools
```

Comment:
Don't use 'sudo' for yum instructions -- it's not configured out-of-the-box on RHEL/CentOS. Use `#` to indicate superuser privs are required.

Reply:
@rkratky  This change is in the README.md file.